csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -5572,6 +5574,33 @@ A distributing party satisfies the "CSAF aggregator" role if the party:
 > CSAF aggregators can be provided for free or as a paid service.
 > To aid in automation, CSAF aggregators MAY mirror CSAF documents from CSAF publishers. Regarding the terms of use they SHOULD consult with the issuing party.
 
+## 7.3 Retrieving rules
+
+The retrieving process splits up into two parts: Finding the `provider-metadata.json` (requirement 7 in section 7.1) and retrieving CSAF documents.
+
+> A retrieving party MAY choose to do the first part only when adding new or updating distributing parties. However, it is recommended to check regulary whether new information is available.
+
+### 7.3.1 Finding provider-metadata.json
+
+**Direct locating**: The following process SHOULD be used to determine the location of a `provider-metadata.json` (requirement 7 in section 7.1) based on the main domain of the issuing party:
+
+1. Checking the Well-known URL (requirement 9 in section 7.1)
+2. Checking the security.txt (requirement 8 in section 7.1)
+3. Checking the DNS path (requirement 10 in section 7.1)
+4. Select one or more `provider-metadata.json` to use.
+
+The first two steps SHOULD be performed in all cases as the security.txt MAY advertise additional `provider-metadata.json`. The thrid step SHOULD only be performed if the first two did not result in the location of at least one `provider-metadata.json`.
```

Comment:
Nit: `thrid ` &rightarrow; `third`

Reply:
It looked weird but I didn't saw what was wrong. :facepalm: